common/logger/src/macros.rs

```diff
@@ -99,6 +99,34 @@ macro_rules! schema {
         )
     };
 
+    // Identifier Keys debug
+    (@ { $(,)* $($out:expr),* }, $($k:ident).+ = ?$val:expr, $($args:tt)*) => {
+        $crate::schema!(
+            @ { $($out),*, &$crate::KeyValue::new($crate::__log_stringify!($($k).+), $crate::Value::from_debug(&format!("{:?}", $val))) },
```

Comment:
You should remove the format macro and the to_string calls here. The values themselves should impl Debug/Display so we shouldn't have to allocate before the call

Reply:
Hmm, I thought I did, but they got themselves back in there when I had to revert because I couldn't figure out what was wrong with the macros